Testing/data/geodesic.txt

```diff
@@ -1 +1 @@
-Geodesic Distance between two points: 1.00833
+Geodesic Distance between two points: 1.64581
```

Comment:
You don't have to fix it since it was already this way, but saving a text file to compare numeric output at least gets a frowny face here.